framework/src/main/java/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -1195,13 +1224,21 @@ protected void analyze(
             if (regularExitStore != null) {
                 regularExitStores.put(method, regularExitStore);
             }
+            Store exceptionalExitStore = analysis.getExceptionalExitStore();
```

Comment:
It looks like that method implementation needs to be improved to me closer to how `getRegularExitStore` is implemented.